### DISCUSSION TITLE
Question about shape rioxarray

### DISCUSSION THREAD

**Discussion by felwind14** [giswqs/geog-312#112] [Q&A] [answered]
Dear professor,

why data_reprojected.shape produces (x,y,band) but data_reprojected.rio.shape drops the band (x,y)

**Comment by giswqs** [comment, marked as answer]
It is probably because the result only has one band, the band is automatically dropped. You can `numpy.expand_dims` to add dimensions if needed. 

https://numpy.org/doc/2.0/reference/generated/numpy.expand_dims.html
